arionBackend/tests/api/test_hierarchy.py

```diff
@@ -6,25 +6,44 @@
 from rest_framework.test import APITestCase
 
 
-class ViewsTestCase(APITestCase):
+class HierarchyTestCase(APITestCase):
+    """
+    This class contains the tests for the Hierarchy API.
+    """
 
     @classmethod
     def setUpTestData(cls):
+        """
+        This method sets up the test class with the required data.
+        """
         pass
 
     def test_get_all_hierarchies(self):
+        """
+        Tests if all hierarchies can be retrieved.
+        """
         request = self.client.get('/api/hierarchy/esper', follow=True)
         self.assertEqual(request.status_code, status.HTTP_200_OK)
 
     def test_get_hierarchy_by_id_success(self):
+        """
+        Tests if one hierarchy can be retrieved via id.
+        """
         request = self.client.get('/api/hierarchy/esper/1', follow=True)
         self.assertEqual(request.status_code, status.HTTP_200_OK)
 
     def test_get_hierarchy_by_id_invalid_id(self):
+        """
+        Tests if the hierarchy cannot be retrieved, if the id is invalid.
+        :return: 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.tommartensen.de/coding_rules#rule_key=python%3AS1131)